syscall_cgo_linux.go

```diff
@@ -13,14 +13,6 @@ import (
 
 var syscall15XABI0 = uintptr(cgo.Syscall15XABI0)
 
-// this is only here to make the assembly files happy :)
-type syscall15Args struct {
-	fn, a1, a2, a3, a4, a5, a6, a7, a8, a9, a10, a11, a12, a13, a14, a15 uintptr
-	f1, f2, f3, f4, f5, f6, f7, f8                                       uintptr
-	r1, r2, err                                                          uintptr
```

Comment:
Wouldn't this information (r1, r2, err) missing?

Reply:
Do you mean the from the error message? func.go doesn't use the field marker but writes everything field unnamed

Reply:
I mean, I was not sure it was really fine to remove these members. The new syscall15Args doesn't have r1, r2, and err, right?

Reply:
They aren't used anywhere. Arguments are now in a1, f1 or R8

Reply:
Thanks!